packages/daemon/src/web-server-node.js

```diff
@@ -138,6 +138,7 @@ export const make = async (_powers, context) => {
         E.sendOnly(webletBootstrap).ping();
 
         // eslint-disable-next-line no-use-before-define
+        await null;
```

Comment:
This should be interfering with the eslint-disable-next-line above it.

Reply:
If it were still using before defining, which it isn't. So I removed this uneeded directive as well.